app/models/statistics_announcement_date.rb

```diff
@@ -21,6 +23,10 @@ def display_date
 
 private
 
+  def update_statistics_announcement_current_release_date
+    statistics_announcement.try(:reload_current_release_date)
```

Comment:
Is `reload_current_release_date` a valid method on `StatisticsAnnouncement`? Github search doesn't find it, is it railsmagic?

Reply:
Rails magic unfortunately, it's created by the `has_one` method: https://api.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html#method-i-has_one